### DISCUSSION TITLE
w5500 Arduino Example

### DISCUSSION THREAD

**Discussion by DeveloperMarcial** [cesanta/mongoose#1854] [Ask questions, get answers] [answered]
**My goal is:** to get the web server to respond with OK when a client connects.
**My actions are:** changing the INO source code.
**The result I see is:** `"READY, IP  M: 10.0.17.0"`
**My expectation & question is:** DHCP does not seem to work. So I tired in the INO file `mif.ip   = 0xb403110a;  //180.3.17.10 <--onstatechange() seems to want backwards IP notation`

Why does `onstatechange()` print `"READY, IP  M: 10.0.17.0"`?
I expect it would print `"READY, IP  M: 10.17.3.180"`

**Comment by DeveloperMarcial** [comment]
The reason DHCP is not working is because the Socket is being opened in `MACRAW` mode (not `TCP` mode) so no IP addy is needed.

So what tool can I use to send a multicast ethernet message to my w5500 MAC addy?

*My previous question stands, however.*

**Comment by scaprile** [comment, marked as answer]
Not being the expert one here on the W5500, I can tell you that this is not a W5500 with Arduino example but a MIP Mongoose example on Arduino with W5500. The whole point of the example is to show how to use MIP, Cesanta's TCP/IP stack. If DHCP does not work for you, please sniff your network (Wireshark) and tell us why.
